tests/test_api.py

```diff
@@ -99,6 +99,19 @@ def test_folder_one_file():
     assert result["tbox"] == ['2018-11-14', '2018-11-14']
 
 
+def test_folder_one_file_details():
+    result = geoextent.fromDirectory('tests/testdata/folders/folder_one_file', bbox=True, tbox=True, details=True)
+    assert "bbox" in result
+    assert "tbox" in result
+    assert "crs" in result
+    assert "details" in result
+    assert result["bbox"] == pytest.approx([7.601680, 51.948814, 7.647256, 51.974624], abs=tolerance)
+    assert result["crs"] == "4326"
+    assert result["tbox"] == ['2018-11-14', '2018-11-14']
+    details = result['details']
+    assert 'muenster_ring_zeit.geojson' in details
```

Comment:
What format is the output here? Should it be valid JSON?

Reply:
The output format is a python dictionary (dictionary of dictionaries). This output could be transform into valid JSON.